requirements/template_non_imported.txt

```diff
@@ -11,3 +11,4 @@ polarion-tools-common
 pre-commit
 pytest-polarion-collect
 pytest-report-parameters
+miq requirement add --extra pytest-mock
```

Comment:
So, this is a command that you need to run on your CLI, which will modify this file, and add `pytest-mock` to it, sorted.

Reply:
I ran the command and pushed the update to requirements/template_non_imported.txt. This is the only file in my repo that was updated. You mention template_scanned.txt  above, but I'm assuming it didn't need to get updated for pytest-mock, but would have been if it was needed.